extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/OpenTelemetryProducer.java

```diff
@@ -154,10 +155,20 @@ private Map<String, String> getOtelConfigs() {
                 if (configValue.getValue() != null) {
                     NameIterator name = new NameIterator(propertyName);
                     name.next();
-                    oTelConfigs.put(name.getName().substring(name.getPosition() + 1), configValue.getValue());
+                    String propertyValue = getConfigPropertyValue(propertyName, configValue);
+                    oTelConfigs.put(name.getName().substring(name.getPosition() + 1), propertyValue);
                 }
             }
         }
         return oTelConfigs;
     }
+
+    private static String getConfigPropertyValue(String propertyName, ConfigValue smallRyeConfig) {
```

Comment:
We will need some tests with values that are currently accepted and others that are failing now.

Reply:
Hi @brunobat. do you mean inside integration-tests repo, or here?

Reply:
they can be simple unit tests related to the `OpenTelemetryUtil` class.

Reply:
perfect, I think that you can review now! 

I changed somethings:
-  `getConfigPropertyValue ` to `normalizeProperty` and 
- `convertDuration` to `normalizeDuration`

I think that is more clear now, if is necessary to normalize another property we add it to the map that I renamed to `OTEL_PROPERTIES_NORMALIZED`. What do you think?